requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Dev requirements
-flake8==5.0.4 # same as super linter
-pre-commit==2.20.0
+flake8==6.0.0 # same as super linter
```

Comment:
Is this still the same version super linter is using?

Reply:
yes
https://github.com/github/super-linter/blob/f0321c38d468cfa3589ed0427a619959fe19d145/dependencies/python/flake8.txt#L1

Reply:
you were right, they released the latest version on October 6, but updated flake version only last week, so we shouldn't update it yet - reverting